ogh/__init__.py

```diff
@@ -3,8 +3,8 @@
                         print_function,
                         unicode_literals)
 
-from ogh.ogh_meta import *  # noqa
-from ogh.ogh import *  # noqa
+from .ogh_meta import *  # noqa
```

Comment:
The explicitness of `ogh.ogh_meta` is better than just `.`

Reply:
@lsetiawan 
I would normally agree, but explicit submodule reference was not building. The prefix 'ogh.' within the ogh/__init__.py file was not being recognized when travis CI builds the test environment.

I referred to Ariel's shablona tutorial to make ogh_meta a relative submodule whenever ogh is called. It is building at least.

Reply:
hm... perhaps not...